src/cryptoadvance/specter/templates/includes/qr-code.html

```diff
@@ -323,20 +356,34 @@
         return;
       }
       this.split(value);
-      if(this.chunks.length == 1){
-        this.isQRlarge = false;
-        this.note.innerHTML = "";
-      }else{
-        this.isQRlarge = true;
-        this.note.innerHTML = "Click to animate";
-      }
-      // we have to use chunks
-      if(this.isQRtoolarge){
-        value = this.chunks[0];
-        this.animate = true;
-      }else{
-        // if this is bc-ur, remove checksum for static QR
+      if(this.mode == 'off'){
         value = this.maybe_remove_checksum(value);
+      }else{
+        if(this.chunks.length == 1){
+          this.isQRlarge = false;
+          this.note.innerHTML = "";
+        }else{
+          this.isQRlarge = true;
+          this.note.innerHTML = "Click to animate";
+        }
+        // we have to use chunks
+        if(this.isQRtoolarge){
+          value = this.chunks[0];
+          this.animate = true;
+        }else{
+          // if this is bc-ur, remove checksum for static QR
+          value = this.maybe_remove_checksum(value);
+        }
+        if(this.mode == 'on'){
```

Comment:
I guess this should also check that the QR gas more than 1 part?

Reply:
Good catch! Fixed.